src/v2/components/Transactions/Detail/Application/index.jsx

```diff
@@ -18,8 +19,10 @@ const Application = ({id, accounts}: TApplication) => {
   const renderAccount = (account, i) => (
     <div className={classes.account}>
       <Label text={`Account ${i + 1}`} hint="" />
-      <Avatar avatarUrl="" name="" width={33} height={33} pubkey={id} />
-      <div className={classes.address}>{account}</div>
+      <Link to={`/account/${id}`} className={classes.accountLink}>
+        <Avatar avatarUrl="" name="" width={33} height={33} pubkey={id} />
+        <div className={classes.address}>{account}</div>
+      </Link>
```

Comment:
@manuel-calavera this looks good, just one background modeling question...

@CriesofCarrots we're treating apps and accounts the same here within transactions, does that matter?

is there a way to disambiguate, or is it program/instruction-specific whether the semantic is account or application?
(let us know if we should treat different program ids differently based on some heuristic)

Reply:
From the chain perspective, a program is an account that has `"executable"=true`. So I think that we can disambiguate that way when loading an actual Application or Account Detail page.

However, currently (elsewhere) in the network explorer, I think that we're identifying applications based on appearance in the `program_id` field in a transaction, correct? (The alternative being tracking LoaderInstruction::Finalize instructions, which set that `executable` byte: https://github.com/solana-labs/solana/blob/bf9c815b9e9d8cf7909a530a611921ebda283fb7/sdk/src/loader_instruction.rs#L22)
If so, I think that for now, we ought to link ids in application fields to `/application/${id}` by default, and we can think about how to handle cases where the transaction-provided account isn't actually a program on the Application Detail page, when we load actual account data. That would mean that on Overview pages, an id's identification as program or application would be instruction specific. Thoughts?